integreat_cms/static/src/js/media-management/component/edit-sidebar.tsx

```diff
@@ -221,6 +229,21 @@ const EditSidebar = ({
                         <label className="secondary my-0">{mediaTranslations.label_file_modified}</label>
                         <p>{file.lastModified}</p>
                     </div>
+                    {file.isGlobal && (
```

Comment:
```suggestion
                    {file.isGlobal && globalEdit && (
```
I would only show the checkbox in the global media library - in the region library, it doesn't make sense since all global files in the region library are not hidden, otherwise they would not be shown.